src/net/discovery/mock.rs

```diff
@@ -0,0 +1,16 @@
+use crate::net::client::MpcHttpConnection;
+use crate::net::discovery::PeerDiscovery;
+
+/// returns valid [`MpcHttpConnection`]s, but not pointing to real servers; only use this if the
+/// clients will never be used. Intended for tests
+pub struct Mock;
```

Comment:
I think we should move it to `test_fixture` module

Reply:
done